packages/components/src/components/Table/Table.scss

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.tableComponent {
+  .bx--table-toolbar {
+    background: transparent;
+  }
+
+  .bx--data-table thead th button.cellText.bx--table-sort {
+    padding: 0;
+  }
+  .noRows {
+    width: 100%;
+    text-align: center;
+    margin-top: 5px;
+    font-style: italic;
+    font-size: 1rem;
+  }
+
+  .bx--batch-actions--active {
+    justify-content: flex-end;
+    visibility: visible !important;
```

Comment:
try to avoid `!important` as it can make it difficult to override later if needed

Reply:
tried and is the only thing working. There are some css bugs where the toolbar and batch actions don't hide when they are not in use. Ill post pictures on the next update.

Reply:
ok thanks